src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -438,16 +451,39 @@ private class SamRecordWriter(output: PathToBam,
 }
 
 /** A writer that writes [[DemuxRecord]]s as [[FastqRecord]]s. */
-private class FastqRecordWriter(prefix: PathPrefix, val pairedEnd: Boolean) extends DemuxWriter {
-  private val writers: IndexedSeq[FastqWriter] = {
-    (if (pairedEnd) Seq("_R1.fastq.gz", "_R2.fastq.gz") else Seq(".fastq.gz"))
-    .map(ext => FastqWriter(Io.toWriter(PathUtil.pathTo(prefix + ext))))
-      .toIndexedSeq
+private[fastq] class FastqRecordWriter(prefix: PathPrefix, val pairedEnd: Boolean, val illuminaStandards: Boolean = false) extends DemuxWriter {
+  private val writers: IndexedSeq[FastqWriter] = extensions.map { ext =>
+    FastqWriter(Io.toWriter(PathUtil.pathTo(prefix + ext)))
+  }.toIndexedSeq
+
+  private[fastq] def extensions: Seq[String] = (pairedEnd, illuminaStandards) match {
+    case (true, true)   => Seq("_R1_001.fastq.gz", "_R2_001.fastq.gz")
+    case (true, false)  => Seq("_R1.fastq.gz", "_R2.fastq.gz")
+    case (false, true)  => throw new IllegalArgumentException("Paired end reads required with illumina-standards")
+    case (false, false) => Seq(".fastq.gz")
+  }
+
+  private[fastq] def readName(rec: DemuxRecord): String = {
+    if (illuminaStandards) {
+      val comment = rec.comment.getOrElse {
```

Comment:
I think all this validation of the incoming read name should at least be mentioned in the usage.  E.g. changing the option to say something like "Require that the input fastq files contain Illumina format read names with comments containing `ReadNum:FilterFlag:0:SampleNumber`, and create output files with illumina format file names."